decidim-budgets/app/packs/stylesheets/budgets.scss

```diff
@@ -69,8 +69,14 @@
     }
   }
 
+  &__projects__container {
+    @apply -mt-12 sm:mt-0;
+  }
+
+
+
```

Comment:
Request for change: 3 new lines in a row, I think you need to run a eslint --fix on these files.